src/token.ts

```diff
@@ -235,6 +235,15 @@ export function getProjectId(crowdinId: string): number {
     return Number(crowdinId.split('__')[1]);
 }
 
+/**
+ *
+ * @param crowdinId crowdin id (from {@link constructCrowdinIdFromJwtPayload})
+ * @returns crowdin user id
+ */
+export function getUserId(crowdinId: string): number {
```

Comment:
I would suggest to write fnc `parseCrowdinId` and it returns you object with 3 fields: `organization`, `projectId` and `userId`